src/services/cache-service.ts

```diff
@@ -73,7 +76,7 @@ export class CacheService {
     /**
      * Returns a list of outdated keys.
      */
-    get outdatedKeys() {
+    get outdatedKeys(): string[] {
```

Comment:
See line comment below.

Reply:
Dunno. It is a "Good Practice" to fully declare functions because without, we can change something seemingly-unrelated in the body and accidentally, silenty, change the signature.

I like following this particular "Good Practice" because (granted, in C++, not yet in a TypeScript project) the same possibility has burned me in the past.